arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -247,13 +247,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/getCredentials": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/listKeys": {
       "post": {
         "tags": [
-          "Registries"
+          "RegistriesGA"
         ],
         "description": "Gets the administrator login credentials for the specified container registry.",
-        "operationId": "Registries_GetCredentials",
+        "operationId": "RegistriesGA_ListKeys",

```

Comment:
"description": "Lists the administrator login keys for the specified container registry."